### PR TITLE
updated the queries to the best of my understanding. Should be in far…

### PR DESCRIPTION
**Pull request by andrewJamesBaker**
… better condition to enter into the data service.

I really just need some more suggestions and advice for writing the queries and updates.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -4,96 +4,119 @@
...
         AND buddy1 != Username.ID
         AND buddy2 != Username.ID
 
 --Login
 ---NEED TO READ: user data: username, password
----NEED TO WRITE: None
-SELECT 
-    FROM
-    ORDER BY 
+---NEED TO WRITE: None, maybe create user upon sign up
```
Comment (kny4) [inline queries.sql:100]: I'm not sure this has anything to do with the schema, but I was thinking of also implementing a "Forgot Password" kinda thing, but it's just an extra idea. I guess we would have to implement some kind of email verification thing first...ooo new research project!

File: queries.sql [hunk 2]
```diff
@@ -4,96 +4,119 @@
...
+    WHERE ID = ${req.params.ID}
 
 --Habit Tracker
 ---NEED TO READ: N/A
 ---NEED TO WRITE: N/A
 SELECT *
-    FROM Habit, UserHabit, HabitCategory, Username
-    WHERE userID = Username.ID
-        AND habitID = Habit.ID
+    FROM
```
Comment (kny4) [inline queries.sql:85]: Huh...how do you store the number of boxes that people have highlighted or whatever?

**Review by kny4 — COMMENTED** [review]
I have no idea what changes to be made, but I'll look into some stuff this weekend!

File: queries.sql (hunk 2 above)
Comment (andrewJamesBaker) [reply queries.sql:85]: I don't really know. Belina would be the person to talk to about that. I don't really know what she wants. Maybe an integer that stores the number of days, but I don't know.

**Pull request merged**